lib/EnsEMBL/REST/Model/ga4gh/references.pm

```diff
@@ -0,0 +1,289 @@
+=head
+Copyright [1999-2014] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+package EnsEMBL::REST::Model::ga4gh::references;
+
+use Moose;
+
+extends 'Catalyst::Model';
+use Data::Dumper;
+with 'Catalyst::Component::InstancePerContext';
+
+has 'context' => (is => 'ro');
+
+use Bio::DB::Fasta;
+use EnsEMBL::REST::Model::ga4gh::ga4gh_utils;
+
+
+our $species = 'homo_sapiens';
```

Comment:
Globals are not very Moosey. Consider an attribute or sub, see discussion [http://www.perlmonks.org/?node_id=690322](url)